clients/privacy-center/package-lock.json

```diff
@@ -1001,31 +1001,6 @@
         "@chakra-ui/theme-tools": "^1.3.1"
       }
     },
-    "node_modules/@chakra-ui/anatomy/node_modules/@chakra-ui/theme-tools": {
```

Comment:
Why are @chakra-ui packages being removed?

Reply:
Okay, what I think is going on here is that Dependabot is just removing this package because it's redundant.

`node_modules/@chakra-ui/anatomy` lists a dependency of `"@chakra-ui/theme-tools": "^1.3.1"` on line 1001, but this dependency is already included down on line 1812.